client/gogeta_client.go

```diff
@@ -13,26 +17,40 @@ import (
 	"github.com/aws/aws-sdk-go/service/sqs"
 )
 
-// GogetaClient is the high level implementation of the gogeta
-// source control system
-type GogetaClient struct {
-	Queue   queuesystem.Messages
-	Log     logger.Log
-	SCM     sourcesystem.SourceSystem
-	Storage storehouse.StoreHouse
+// Gogeta is the source control manager implementation
+type Gogeta struct {
```

Comment:
Data doesn't need to be exposed, but the systems need to be exposed so we can mock out / reuse the GogetaClient (specify outside of this scope)